src/selectize.js

```diff
@@ -648,12 +649,14 @@ $.extend(Selectize.prototype, {
 
 		if (self.ignoreFocus) {
 			return;
-		} else if (!self.ignoreBlur && document.activeElement === self.$dropdown_content[0]) {
-			// necessary to prevent IE closing the dropdown when the scrollbar is clicked
-			self.ignoreBlur = true;
-			self.onFocus(e);
-			return;
 		}
+		// Bug fix do not blur dropdown here
```

Comment:
This code causes the drop down to flash open then closed when a click is detected outside of the control.  Let's remove it - nobody uses IE11 any more.